package.json

```diff
@@ -57,13 +57,13 @@
     "eslint": "^1.7.3",
     "eslint-config-gulp": "^2.0.0",
     "expect": "^1.20.2",
-    "gulp": "gulpjs/gulp#4.0",
+    "gulp": "^4.0.0",
     "gulp-test-tools": "^0.6.1",
-    "jscs": "^2.3.5",
+    "jscs": "^3.0.7",
```

Comment:
I thought jscs changed a bunch of stuff in 3.0 that made our config file invalid?

Reply:
[Release note of jscs 3.0](https://github.com/jscs-dev/node-jscs/releases/tag/v3.0.0) wrote:
> We didn't rename/remove any rules. ...

So I think there are no problem to use 3.0.